package/src/index.ts

```diff
@@ -0,0 +1,537 @@
+                ? `?${cur}=${value}`
+                : acc + `&${cur}=${value}`
+        }, "") : ""
+
+        // Main request
+        const req = await axios({
+            url: encodeURI(`${baseUrl}/${url}${queryParams}`),
+            responseType: "json",
+            headers: this.token
+                ? { Authorization: this.token, "User-Agent": "unofficial-valorant-api/node.js/2.3.0", }
+                : { "User-Agent": "unofficial-valorant-api/node.js/2.3.0", },
+            ...config
+        }).catch(error => error)
+
+        // Formatting response
+        return {
+            status: req.response ? req.response.status : req.status,
+            data: req.response ? null : !req.config.headers["Content-Type"] ? this._parseBody(req.data) : req.data,
+            rateLimits: {
+                used: Number(req.response ? req.response.headers["x-ratelimit-limit"] : req.headers["x-ratelimit-limit"]),
+                remaining: Number(req.response ? req.response.headers["x-ratelimit-remaining"] : req.headers["x-ratelimit-remaining"]),
+                reset: Number(req.response ? req.response.headers["x-ratelimit-reset"] : req.headers["x-ratelimit-reset"]),
+            },
+            error: req.response ? this._parseBody(req.response.data) : null,
+            url: req.config.url
+        }
+    }
+
+    /**
+	 * @internal
+	 * Make sure each key has a non empty value
+	 * @param input - Object to check for non empty values
+	 * @throws {@link TypeError} - Only if there is a empty value to a key
+	 */
+    private async _validateArgs(input: { [key: string]: any }) {
+        Object.keys(input).forEach(key => {
+            if (input[key] === null || input[key] === undefined) throw new TypeError(`Missing parameter: "${key}"`)
+        })
+    }
+
+    /**
+	 * Get an image of a valorant crosshair by its code
+	 * @example
+	 * Load and write a crosshair to crosshair.png
+	 * ```js
+	 * import { writeFileSync } from "fs"
+	 * 
+	 * const crosshair = await default.getCrosshair("0;s;1;P;c;5;o;1;d;1;z;3;0b;0;1b;0;S;c;4;o;1")
+	 * writeFileSync("crosshair.png", crosshair.data)
+	 * ```
+	 * @param code - Valorant crosshair code
+	 * @param size - (optional) Image size (default: `1024`)
+	 * @returns The image of the crosshair as a {@link Buffer}
+	 */
+    async getCrosshair(code: string, size?: number): Promise<APIResponse<CrosshairResponse>> {
+        this._validateArgs({ code })
+        return this._fetch<CrosshairResponse>("v1/crosshair/generate", { id: code, size }, {
+            responseType: "arraybuffer"
+        })
+    }
+
+    /**
+	 * Get all announcements from the valorant website of a country
+	 * @param countryCode - Country code of website
+	 * @returns List of announcements from the valorant website
+	 */
+    async getAnnouncements(countryCode: "en-us" | "en-gb" | "de-de" | "es-es" | "es-mx" | "fr-fr" | "it-it" | "ja-jp" | "ko-kr" | "pt-br" | "ru-ru" | "tr-tr" | "vi-vn"): Promise<APIResponse<WebsiteResponse>> {
```

Comment:
calling this announcements will confuse users. As they only see website in swagger hub documentation, and alsou your response is called `WebsiteResponse`.

Reply:
I feel it would do the opposite as I originally thought it just gave a link to the countries valorant website and only saw it was announcements after looking at the response. I call it `WebsiteResponse` because I want to keep the type names consistent with the actual API endpoint, but I will change it back

Reply:
Yeah but everyone sees the swagger doc and searches for the website endpoint but can't find it (on the first look).